src/snowflake/connector/aio/_cursor.py

```diff
@@ -1087,5 +1088,11 @@ async def query_result(self, qid: str) -> SnowflakeCursor:
         return self
 
 
-class DictCursor(DictCursorSync, SnowflakeCursor):
-    pass
+class DictCursor(SnowflakeCursor):
+    """Cursor returning results in a dictionary."""
+
+    def __init__(self, connection) -> None:
+        super().__init__(
+            connection,
+            use_dict_result=True,
+        )
```

Comment:
curious what's the purpose of change here?

Reply:
DictCursor will return a dict result instead of regular one. It is used in some of the tests.
I think only put a `pass` in the function body is not enough to realize that purpose? It needs to set `use_dict_result` = True.